app/forms/decidim/time_tracker/admin/activity_form.rb

```diff
@@ -10,8 +10,8 @@ class ActivityForm < Decidim::Form
         translatable_attribute :description, String
 
         attribute :active, Boolean
-        attribute :start_date, Decidim::Attributes::LocalizedDate
-        attribute :end_date, Decidim::Attributes::LocalizedDate
+        attribute :start_date, Decidim::Attributes::TimeWithZone
+        attribute :end_date, Decidim::Attributes::TimeWithZone
```

Comment:
❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️

Reply:
It was horrible to deal with this when editing dates